assets/sass/_all-digital.scss

```diff
@@ -3,12 +3,6 @@
 // external libraries and frameworks
 @import 'all-digital/vendors';
 
-// Utilities
```

Comment:
keep this where it was, shame should still be able to trump everything

Reply:
looking at this again, the utils is for mixins and stuff so it needs to be first, its just poorly named.
Let's create a new "utility" file that includes the helpers? or maybe we can call it helpers?